vega_sim/scenario/common/agents.py

```diff
@@ -79,14 +82,25 @@ def initialise(self, vega: VegaServiceNull):
             asset=self.asset_id,
             amount=self.initial_asset_mint,
         )
-        self.vega.wait_fn(2)
+        self.vega.wait_fn(5)
+        self.vega.wait_for_total_catchup()
+
+        self.vega.submit_simple_liquidity(
+            self.wallet_name,
+            self.market_id,
+            1000,
+            "0.0015",
+            vega_protos.PEGGED_REFERENCE_BEST_BID,
+            vega_protos.PEGGED_REFERENCE_BEST_ASK,
+            1,
+            1,
+        )
```

Comment:
I know there is no impact on trades, but could I ask why we need the market order trader post liquidity here?

Reply:
Ah, we definitely don't. Was using this branch for some custom simulation so missed cutting some bits out, thanks for catching :+1: